packages/palette/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@leafygreen-ui/palette",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Palettes",
+	"main": "./dist/index.js",
+	"types": "./dist/index.d.ts",
+  "scripts": {
+		"build": "../../node_modules/.bin/webpack --config ../../webpack.config.js",
+		"ts:emit": "tsc --project ./tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@leafygreen-ui/lib": "^2.1.7"
```

Comment:
What do we need lib for?

Reply:
Emotion and such is used for the Storybook entry.

Reply:
So devDependency?  But aren't those in the root `package.json` so no need to have it explicitly here?

Reply:
That's fair enough. I actually will need to add it to the root `package.json` since it's not there yet.